packages/forms-web-app/src/dynamic-forms/dynamic-components/list-add-more/question.js

```diff
@@ -303,8 +304,7 @@ class ListAddMoreQuestion extends Question {
 		}
 
 		// move to the next question
-		const updatedJourney = new journey.constructor(journeyResponse);
```

Comment:
well I'm glad these can go

Reply:
I'm not sure these were ever needed? the journey coming in was always made with the latest response